internal/driver/driver.go

```diff
@@ -165,7 +167,71 @@ func (c *Conn) WriteRaw(b []byte) error {
 }
 
 // Request sends the given request to the connection and returns the response.
+// If header MessageLength or RequestID is not specified, it assings the proper values.
+// For header.OpCode the wire.OpCodeMsg is used as default.
 func (c *Conn) Request(ctx context.Context, header *wire.MsgHeader, body wire.MsgBody) (*wire.MsgHeader, wire.MsgBody, error) {
```

Comment:
I think we should also document what kind of errors are returned. For example, response with `{"ok": 0}` will not be returned as an error